terraform/domains/infrastructure/config/zones_Terrafile

```diff
@@ -0,0 +1,3 @@
+domains:
+    source:  "https://github.com/DFE-Digital/terraform-modules"
+    version: "testing"
```

Comment:
Can we use stable for the zone? It can't be tested in a different environment so it should be as stable as possible.